### ISSUE TITLE
subkey(...) doesn't work

### ISSUE THREAD

**Issue by 7ERr0r** [ColinFinck/nt-hive#1]
Hi, thanks for the awesome crate.

Currently, `.subkey()` doesn't work.
https://github.com/7ERr0r/winkeyhive/blob/09af968cafabf71edcb3c74841864249a9633b62/src/main.rs#L129

Also it would be great if you could read hives after the Windows hibernates.
Even though the NTFS is read-only, and the `SOFTWARE` file perfectly readable:
```
Searching: /media/ubuntu/3AF62122F620E03D
    Found: /media/ubuntu/3AF62122F620E03D/Windows/System32/config/SOFTWARE
Error: "Error parsing hive file: The sequence numbers in the base block do not match (1746 != 1745)"
```

The program stops in the validation stage.
https://github.com/ColinFinck/nt-hive/blob/master/src/hive.rs#L197

**Comment by ColinFinck** [comment]
Hi!

> Currently, `.subkey()` doesn't work. https://github.com/7ERr0r/winkeyhive/blob/main/src/main.rs#L129

`.subkey()` only looks for a single subkey of the current key.
If you want to pass a subpath of nested keys, please check out [`.subpath()`](https://docs.rs/nt-hive/0.1.0/nt_hive/struct.KeyNode.html#method.subpath)


> Also it would be great if you could read hives after the Windows hibernates. Even though the NTFS is read-only, and the `SOFTWARE` file perfectly readable:
> 
> ```
> Searching: /media/ubuntu/3AF62122F620E03D
>     Found: /media/ubuntu/3AF62122F620E03D/Windows/System32/config/SOFTWARE
> Error: "Error parsing hive file: The sequence numbers in the base block do not match (1746 != 1745)"
> ```
> 
> The program stops in the validation stage. https://github.com/ColinFinck/nt-hive/blob/master/src/hive.rs#L197

I had thought about that, but didn't have a use-case back then.
Reading hives after Windows has hibernated may be a valid use-case though (even though that is risky for the modifications that haven't been flushed to disk yet).
I'll try to come up with something. For the moment, you are free to just comment out the `hive.validate()?` call on hive.rs:96

**Comment by 7ERr0r** [comment]
Thanks for response!

Of course it's `subpath()`, my bad.

I was guided by an example of `.subkey()` btw.
https://github.com/ColinFinck/nt-hive/blob/master/README.md?plain=1#L43

**Comment by ColinFinck** [comment]
Your bug report was totally valid! There shouldn't be a mistake in the one example I give in the README file.
I have just fixed that and added `Hive::without_validation` to enable your use-case.
I also released nt-hive 0.2.0 along with some other fixes (check https://github.com/ColinFinck/nt-hive/blob/master/CHANGELOG.md)

Hope the new version works better for you :)
